<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TasksDock</class>
 <widget class="QDockWidget" name="TasksDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="edit-find-replace">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="windowTitle">
   <string>Tasks</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="5">
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>384</width>
         <height>239</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="scrollLayout"/>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="addButton">
      <property name="text">
       <string>Add task</string>
      </property>
      <property name="icon">
       <iconset theme="list-add">
        <normaloff>.</normaloff>.</iconset>
      </property>
      <property name="popupMode">
       <enum>QToolButton::InstantPopup</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="saveButton">
      <property name="text">
       <string>Export</string>
      </property>
      <property name="icon">
       <iconset theme="document-save">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="loadButton">
      <property name="text">
       <string>Import</string>
      </property>
      <property name="icon">
       <iconset theme="document-open">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>269</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4">
     <widget class="QToolButton" name="runButton">
      <property name="text">
       <string>Run</string>
      </property>
      <property name="icon">
       <iconset theme="system-run">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
